core/Commands/TemplateCommand.cs

```diff
@@ -252,7 +251,7 @@ protected IEnumerable<string> ParseConfigFile(IFileInfo configFile)
                 args.AddRange(new string[] { "--DBServerODS", configJson["Package[Product.Database.Ods].Database.Server"]?.Value ?? configJson["DATABASE_ODS_MSSQL_ADDRESS"]?.Value ?? "" });
                 args.AddRange(new string[] { "--DBServerDWH", configJson["Package[Product.Database.Dwh].Database.Server"]?.Value ?? configJson["DATABASE_DWH_MSSQL_ADDRESS"]?.Value ?? "" });
                 args.AddRange(new string[] { "--ReportServerURI", configJson["Package.ReportingServices.Address"]?.Value ?? configJson["REPORTING_SSRS_WEB_PORTAL_URL"]?.Value ?? "" });
-                if (configJson["Product.Database.IsAlwaysOn"]?.Value ?? bool.Parse(configJson["DATABASE_MSSQL_ALWAYS_ON_ENABLED"]?.Value ?? false))
+                if (configJson["Product.Database.IsAlwaysOn"]?.Value ?? bool.Parse((configJson["DATABASE_MSSQL_ALWAYS_ON_ENABLED"]?.Value ?? false).ToString()))
```

Comment:
we should not be relying on the ToString() behaviour here, I'm not sure if this is an implementation detail. If the Value is an object, we should extract the Value property from it. We could also implement an extension method for this, if the Value is a simple value just return it, otherwise return .Value